doc/mgr/dashboard.rst

```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
   <https://prometheus.io/docs/operating/security/>` for more detailed
   information.
 
-Grafana and Prometheus are likely going to be bundled and installed by some
-orchestration tools along Ceph in the near future, but currently, you will have
-to install and configure both manually. After you have installed Prometheus and
-Grafana on your preferred hosts, proceed with the following steps.
+Installation and Configuration using cephadm
+""""""""""""""""""""""""""""""""""""""""""""
 
-1. Enable the Ceph Exporter which comes as Ceph Manager module by running::
+Grafana and Prometheus can be installed using :ref:`cephadm`. It will
```

Comment:
What do you mean by "it" here? The Dashboard?

Reply:
Good spotting :sweat_smile: Both components will automatically be configured by cephadm. I'll adapt it to `they`.